worlds/mlss/docs/en_Mario & Luigi Superstar Saga.md

```diff
@@ -0,0 +1,56 @@
+# Mario & Luigi: Superstar Saga
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and
+export a config file.
+
+## What does randomization do to this game?
+
+Items which the player would normally acquire throughout the game have been moved around. Logic remains, so the game is
+always able to be completed, but because of the item shuffle, the player may need to access certain areas before they
+would in the vanilla game.
+
+The game has been changed to an open-world style as opposed to the linear style the vanilla game has.
+
+Other Features such as Turbo through textboxes (Hold L/R+A) and Pipe Warping from any room (Hold L+R+SELECT) have been added for convenience.
+
+Enemies and Bosses can be randomized, and their stats can be scaled to feel more like the vanilla game's stats.
+
+Other aspects of the game can be randomized as well such as music, sounds, battle backgrounds, Mario and Luigis Colors, and more.
+
+## What is the goal of Mario & Luigi: Superstar Saga when randomized?
+
+Defeat Cackletta's Soul in Bowser's Castle. This requires you to collect all 4 beanstar pieces and restore the beanstar, as well as Peach's Extra Dress and the Fake beanstar, and bring them to Fawful at the end of Jokes End. This also requires Ultra Hammers, Thunder Hand and Green Pearl Bean, Fire Hand and Red Pearl Bean, Green Goblet, The Beanbean Brooch, All 3 Chuckola Fruits, Peasley's Rose, and either the Membership card or Red Goblet.
```

Comment:
```suggestion
Defeat Cackletta's Soul in Bowser's Castle. This requires you to collect all 4 Beanstar Pieces, restore the Beanstar, and bring Peach's Extra Dress and the Fake Beanstar to Fawful at the end of Jokes End. 

In total, this requires:
- 4 Beanstar Pieces
- Peach's Extra Dress
- Fake Beanstar
- Ultra Hammers
- Fire Hand
- Thunder Hand
- Red Pearl Bean
- Green Pearl Bean
- Green Goblet
- Peasley's Rose
- Beanbean Brooch
- All 3 Chuckola Fruits
- Membership Card OR Red Goblet
```
This section was a bit tough to read when playing and trying to determine what I still need to complete the game. Here's a suggestion for how to better format this section.